tests/post/steps/test_volume.py

```diff
@@ -94,6 +99,22 @@ def _check_status(status='Available'):
         name='checking status of Volume {}'.format(name)
     )
 
+
+
+@given(parsers.parse("I create the following object:\n{body}"))
+def create_pvc(host, body, k8s_client):
+    k8s_client.create_namespaced_persistent_volume_claim(
+        namespace='monitoring',
+        body=yaml.safe_load(body)
+    )
+
+
+@given(parsers.parse("I create the following Pod:\n{body}"))
```

Comment:
Same remarks, either:
- `@given(parsers.parse("the following Pod exist:\n{body}"))`
- `@then(parsers.parse("I create the following Pod:\n{body}"))`